### PR TITLE
Hourofcode.com front page banner style changes

### PR DESCRIPTION
**Pull request by caleybrock**
Updates to the home page banner.

### CODE REVIEW

**Comment by caleybrock** [comment]
screen shot 2015-09-17 at 2 58 28 pm

I changed the positioning of the call to action like we talked about and picked some nicer photos.

On mobile this moves the calls to action down pretty far, so I've decided to keep mobile UI as is originally.

please review: @breville 
cc: @tanyaparker

**Comment by ashercodeorg** [comment]
Take my opinion with a grain of salt... But I prefer the aesthetics of the blue background over its elimination.

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css [hunk 1]
```diff
@@ -53,3 +54,44 @@
...
+  margin-top: 0px;
+  margin-bottom:10px;
+  padding:20px
+}
+
+#leftbutton {
+  position:absolute;
+  left:0;
+  top:0;
+  text-align:center;
```
Comment (breville) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: we should be consistent on whether there's a space after the colon or not.  i think there should be.

**Comment by breville** [comment]
I wonder if we should move those big banner images to /shared so they can be shared between hourofcode.com and code.org rather than being duplicated?  might be interesting to look into if you have time.

**Comment by breville** [comment]
looks good.  you might want to ask Alice to give it a once-over before pushing live.

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css (hunk 1 above)
Comment (caleybrock) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: oops.. yes I agree! This was a product of me copying from the inline styles.

**Comment by caleybrock** [comment]
@breville the banner images can be shared, but I'm going to make a todo for myself to do in a separate PR. I think it will take 2 separate PRs plus a gsheet edit to complete.

**Comment by breville** [comment]
that sounds like a great plan.  thanks.

File: pegasus/sites.v3/hourofcode.com/public/css/front-page.css (hunk 1 above)
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/css/front-page.css]: I agree.

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 2]
```diff
@@ -51,17 +51,17 @@ social:
...
 
-  #fullwidth3{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/girl-learning.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
+  #fullwidth3{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_3.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 50%"}
 
-  #fullwidth4{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/kids-learning.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 30% 40%"}
+  #fullwidth4{style: "z-index: -5; position:absolute; left: 0; top: 0; background-image: url(#{localized_image('/images/homepage_4.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 50% 10%;"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/index.haml]: Generally a space between the attribute and value, but if it's just 0 I think no space is fine. I'm going to fix this one that happens to be different from the rest and all the "top" attributes which have a space. Just for consistency.

Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/index.haml]: Ah, whatever I'll just give em all spaces.

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 3]
```diff
@@ -51,17 +51,17 @@ social:
...
 
-  #fullwidth5{style: "z-index: -5; position:absolute; left: 0; top: 0; background-image: url(#{localized_image('/images/boy-certificate.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 40% 50%"}
+  #fullwidth5{style: "z-index: -5; position:absolute; left:0; top: 0; background-image: url(#{localized_image('/images/homepage_5.jpg')}); background-repeat: no-repeat; background-size: cover; width: 100%; height:100%; opacity: 0; background-position: 30% 40%"}
 
   #fullwidth{style: "/*position:absolute; left:0; top: 0; */width: 100%; /*height:#{imageheight}px*/"}
```
Comment (tanyaparker) [inline pegasus/sites.v3/hourofcode.com/public/index.haml]: Should I delete this?

**Comment by tanyaparker** [comment]
LGTM all tests passed

**Comment by tanyaparker** [comment]
I'll look at updating HAML linter.

**Pull request merged**